sw/host/opentitanlib/src/otp/alert_handler.rs

```diff
@@ -275,10 +275,10 @@ impl AlertRegs {
         match config.enabled {
             AlertEnable::None => {}
             AlertEnable::Enabled => {
-                reg_bits.set(ALERT_HANDLER_CLASSA_CTRL_SHADOWED_LOCK_BIT as usize, true);
                 reg_bits.set(ALERT_HANDLER_CLASSA_CTRL_SHADOWED_EN_BIT as usize, true);
             }
             AlertEnable::Locked => {
+                reg_bits.set(ALERT_HANDLER_CLASSA_CTRL_SHADOWED_LOCK_BIT as usize, true);
```

Comment:
For reference, [this](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/lib/drivers/alert.c;drc=b4d0de8f261f6397c4af74d33b0dc7afae73e224;l=169) is the corresponding line from the alert handler driver.